### PR TITLE
Eureka Translation fr

### PR DESCRIPTION
**Pull request by VanSlanzar**
Translation and adjustement correction fr

### CODE REVIEW

File: ui/eureka/eureka.js [hunk 1]
```diff
@@ -477,7 +477,7 @@ let Options = {
           trackerName: {
             en: 'Strider',
             de: 'Läufer',
-            fr: 'Trotteur',
+            fr: 'Simurgh',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: This is incorrect, I think.  The tracker name here is the name from https://ffxiv-eureka.com/.  Start a tracker, click on the green clipboard icon, and you'll get something like `NMs dont la fenêtre est disponible : Pampa → Seigneur → Teles → Empereur → Callisto → Number → Jaha → Amemet → Caym → Bomba → Serket → Julika → Cavalier → Poly → Trotteur → Hazmat → Fafnir → Amarok → Lamashtu → Pazuzu (Vents violents dans 37m)`

In this case, Trotteur is the name.

File: ui/eureka/eureka.js [hunk 2]
```diff
@@ -739,23 +739,23 @@ let Options = {
...
+            fr: 'Graboïde',
             ja: 'グラヴォイド',
             ko: '그라보이드',
             cn: '异形魔虫',
           },
           trackerName: {
             en: 'Glavoid',
             de: 'Glavoid',
-            fr: 'Glaboïde',
+            fr: 'Graboïde',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: This is also incorrect, based on tracker name.

File: ui/eureka/eureka.js [hunk 3]
```diff
@@ -1166,15 +1166,15 @@ let Options = {
...
+            fr: 'Leucosie',
             ja: 'レウコシアー',
             cn: '琉科西亚',
             ko: '레우코시아',
           },
           trackerName: {
             en: 'Leucosia',
             de: 'Leucosia',
-            fr: 'Leucosia',
+            fr: 'Leucosie',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: This is not the tracker name either.

File: ui/eureka/eureka.js [hunk 4]
```diff
@@ -1271,23 +1271,23 @@ let Options = {
...
+            fr: 'Ascalaphe',
             ja: 'アスカラポス',
             cn: '阿斯卡拉福斯',
             ko: '아스칼라포스',
           },
           trackerName: {
             en: 'Owl',
             de: 'Askala',
-            fr: 'Askala',
+            fr: 'Ascala',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: This too.

File: ui/eureka/eureka.js [hunk 5]
```diff
@@ -1300,23 +1300,23 @@ let Options = {
...
+            fr: 'Batime Le Grand Duc',
             ja: 'グランドデューク・パティム',
             cn: '巴钦大公爵',
             ko: '바팀 대공',
           },
           trackerName: {
             en: 'Batym',
             de: 'Batym',
-            fr: 'Duc Batym',
+            fr: 'Batime',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: And this

File: ui/eureka/eureka.js [hunk 6]
```diff
@@ -1329,23 +1329,23 @@ let Options = {
...
+            fr: 'Aetolos',
             ja: 'アイトロス',
             cn: '埃托洛斯',
             ko: '아이톨로스',
           },
           trackerName: {
             en: 'Aetolus',
             de: 'Aetolus',
-            fr: 'Aetolus',
+            fr: 'Aetolos',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: And this

File: ui/eureka/eureka.js [hunk 7]
```diff
@@ -1498,23 +1498,23 @@ let Options = {
...
+            fr: 'Saule Pleureur',
             ja: 'ランバージャック・デスマッチ',
             cn: '垂柳树人',
             ko: '럼버잭',
           },
           trackerName: {
             en: 'Jack',
             de: 'Holz',
-            fr: 'Jack',
+            fr: 'Bûcheron',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: And this

**Comment by VanSlanzar** [comment]
i' use google trad waitng plz. 
Trackers don't have the right names. I did the 4 eureka and I used my personal js on act. That's why I came to see you. To change things.

**Comment by VanSlanzar** [comment]
ah i understand, it's difference mobname and name trackername correctly ? i go change this, sorry.
all mobname are correct, just i change tracker name ?

**Comment by quisquous** [comment]
> ah i understand, it's difference mobname and name trackername correctly ? i go change this, sorry.
> all mobname are correct, just i change tracker name ?

Sorry for the lack of documentation, but:
* label is what shows on the map, and is just visual
* mobname is the actual name of the mob in game, used for figuring out if something is alive or not, this may go away with #903 
* trackername is the name from the ffxiv-eureka.com tracker, so that you can paste a list of mobs that are available and have the map update

Yeah, I think if you remove all the trackername changes, this should be ok.  I did not check if the mob names were correct or not.  You can always verify with https://github.com/ufx/SaintCoinach locally.

**Comment by VanSlanzar** [comment]
Ok, i've verified all name. Mobname is really name in game French. For trackername, it's name in the tracker.
https://ffxiv-eureka.com/. I'm done. ^^. I hope it's ok.

File: ui/eureka/eureka.js [hunk 8]
```diff
@@ -1900,7 +1900,7 @@ let Options = {
           },
           trackerName: {
             en: 'Barong',
-            de: 'Baron',
+            de: 'Barong',
```
Comment (quisquous) [inline ui/eureka/eureka.js]: Double checked this one, and you're right that it is Barong.  Thanks!

Comment (VanSlanzar) [reply ui/eureka/eureka.js]: arf i correct

Comment (Akurosia) [reply ui/eureka/eureka.js]: The Name of the Mob is "Baron Feuermähne" and not Barong in german

**Pull request merged**